monitors/SyntheticPegMonitor.js

```diff
@@ -39,16 +40,32 @@ class SyntheticPegMonitor {
         isValid: x => {
           return toBN(x).lte(toBN(toWei("100"))) && toBN(x).gte(toBN("0"));
         }
+      },
+      volatilityWindow: {
+        // `volatilityWindow`: Length of time (in seconds) to snapshot volatility.
+        value: 60 * 60, // 1 hour.
+        isValid: x => {
+          return x >= 0;
+        }
+      },
+      volatilityAlertThreshold: {
+        // `volatilityAlertThreshold`: Error threshold for pricefeed's price volatility over `volatilityWindow`.
+        // Expressed as a %.
```

Comment:
Is it expressed as a %? the `deviationAlertThreshold` is not expressed as a percentage; 0.2 is used to represent 20% and in your `volatilityAlertThreshold` the 0.05 is used to represent 5%. This comment is a bit misleading then.

Reply:
got it i'll fix